packages/sdk-react/src/core/queries/useReceivables.ts

```diff
@@ -163,6 +163,11 @@ export const useIssueReceivableById = () => {
       queryClient.setQueryData(receivablesQueryKeys.detail(id), receivable);
       invalidate();
 
+      /** We have to invalidate a PDF query to get the latest PDF */
+      queryClient.invalidateQueries({
```

Comment:
optional: if you have time, you could add invalidation to `useEntityCache`, as it is implemented in `useEntityListCache`, and re-use it here.

Reply:
Done: [f549c7e](https://github.com/team-monite/monite-sdk/pull/87/commits/f549c7eb750cab6c458851bda38c1096bda182ae)

Reply:
In my opinion `useReceivablePdfListCache` is overcomplicating.

Reply:
Moved it back: [289e24e](https://github.com/team-monite/monite-sdk/pull/87/commits/289e24e46beae79fbab15a4d0d0b167deed7b20b)

Reply:
Invalidation keeps the previous query in the cache. When we update the Invoice, probably, the previous result for the PDF must be not just invalidated, but reset, to show spinner immediately instead of the old PDF.

Reply:
So then I think it's better to have a separate function to handle those query client requests
[c1e18c8](https://github.com/team-monite/monite-sdk/pull/87/commits/c1e18c8e7e7434340e198d3b482c35ef15e7d9e6)